src/modules/tests/test-data/test-data.models.ts

```diff
@@ -5,8 +5,8 @@ export type FaultPayload = {
   newFaultCount: number,
 };
 
-// This type is used for test report validation
-// TODO: Delete this type and use the one from the api-definitions
+export type LegalRequirements = CatBLegalRequirements | CatBELegalRequirements;
```

Comment:
How come we're choosing not to do what was outlined in the TODO?

Reply:
I spoke with szabi and there is nothing we can do with this TODO